src/main/webapp/assets/js/feedScript.js

```diff
@@ -644,28 +658,47 @@ async function addPosts(posts) {
     postCard.setAttribute('id', post.id);
     postCard.setAttribute('tabindex', '0');
 
+    // Adds the image from the image serving URL.
+    // If there is not a valid URL, then display the stock SVG.
+    const cardImage = document.createElement('img');
+    cardImage.setAttribute('class', 'card-image');
+    let imageSource = post.imageServingUrl;
+    if (imageSource === 'no image') {
+      imageSource = './assets/svg/forkandknife.svg';
```

Comment:
Let's try to find/create a stock image that matches the current theme. The app already looks really good and would like to keep that style.

Reply:
Good idea, I just found another stock image that is a bit cuter. 
@areetaw @mirrorkeydev if you guys have an image in mind I'd love to use it!
![Screenshot 2020-08-30 at 8 59 19 PM](https://user-images.githubusercontent.com/48428336/91681596-c0c84100-eb03-11ea-98c5-05e5dd177670.png)

Reply:
ooh that's really cute!